tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -42,6 +42,9 @@ object Discover {
     path
   }
 
+  def rustc: Path = discover("rustc", "RUSTC")
+  def cargo: Path = discover("cargo", "CARGO_HOME")
```

Comment:
If the rust integration is extracted, I guess these should be moved/removed as well.